store-implementation/parquet-store/src/main/java/uk/gov/gchq/gaffer/parquetstore/partitioner/GroupPartitioner.java

```diff
@@ -87,16 +89,17 @@ public int getPartitionId(final PartitionKey partitionKey) {
      * @return a list of all the partition ids which could contain data matching the provided partial key
      */
     public List<Integer> getPartitionIds(final Object[] partialKey) {
+        final int partitionKeyLength = splitPoints.isEmpty() ? 0 : splitPoints.get(0).getLength();
         if (null == partialKey) {
             throw new IllegalArgumentException("getPartitionIds cannot be called with null partialKey");
         }
         if (0 == splitPoints.size()) {
```

Comment:
Can this just be changed to if(splitPoints.isEmpty).  This is a safer way to check an empty List.

Reply:
I have updated for both this and the above comment